treeherder/etl/runnable_jobs.py

```diff
@@ -123,6 +123,9 @@ def _taskcluster_runnable_jobs(project, decision_task_id):
     tc_graph = {}
     if not decision_task_id:
         decision_task_id = _query_latest_gecko_decision_task_id(project)
+    # If we still don't have task id, lets just bail out now
```

Comment:
This is a reasonable thing to do, but the comment makes it look like a hack. I would reword this to something like:

```py
# Some trees (e.g. comm-central) don't have a decision task, which means there are no taskcluster runnable jobs
```